src/commonMain/kotlin/com/epam/drill/Constants.kt

```diff
@@ -21,3 +21,6 @@ const val SYSTEM_CONFIG_PATH = "DRILL_AGENT_CONFIG_PATH"
 const val SYSTEM_HTTP_HOOK_ENABLED = "DRILL_HTTP_HOOK_ENABLED"
 const val SYSTEM_JAVA_APP_JAR = "JAVA_APP_JAR"
 const val HTTP_HOOK_ENABLED = "drill.http.hook.enabled"
+
+const val KAFKA_PRODUCER_INTERFACE = "org/apache/kafka/clients/producer/Producer"
+const val KAFKA_CONSUMER_SPRING = "org/springframework/kafka/listener/KafkaMessageListenerContainer\$ListenerConsumer"
```

Comment:
Will this work for the consumer on [org.apache.kafka](https://mvnrepository.com/artifact/org.apache.kafka) and not only on [springframework.kafka](https://mvnrepository.com/artifact/org.springframework.kafka) ?

Reply:
it will work only for  springframework.kafka

Reply:
I mean, do we need org.apache.kafka support?

Reply:
for current needs no. In the future, it will need to implement